files/openmw.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Alexandre Moine <nobrakal@gmail.com> -->
+<application>
+ <id type="desktop">openmw.desktop</id>
+ <metadata_license>CC0-1.0</metadata_license>
```

Comment:
For example, in France, my country, I am obliged to name myself, because a totally free license (like CC-0) is not legal. But the license still CC-0, even with my name :)